frontend/src/components/contact/contact_us.js

```diff
@@ -1,5 +1,65 @@
-import React from 'react';
+import React from "react";
+import { Button2 } from "../util/button/button";
+import "./contact_us.css";
+const ContactUs = () => {
+  return (
+    <div class="contact-section">
+      <div class="contact-parent">
+        <div class="contact-child child1">
+          <img src="./images/contact.jpg" alt="" className="contact-image" />
+        </div>
 
-const ContactUs = () => <h1 className='contact-us'>CONTACT US</h1>;
+        <div class="contact-child child2">
+          <h3 class="contact-header-text">Get in touch</h3>
```

Comment:
can we please have this heading in the center of the right sided form

Reply:
Yes I have updated the requested changes in the recent commit.